js/ui/menu.d.ts

```diff
@@ -277,6 +277,17 @@ export interface dxMenuItem extends dxMenuBaseItem {
      * @type Array<dxMenuItem>
      */
     items?: Array<Item>;
+    /**
+     * @docid
+     * @public
+     */
+    url?: string;
```

Comment:
This functionality we support only in menu widget. But it is not work correct if it moved to ui.menu.js file. So it live in the menuBase. But why we need to declare comments for dxMenuBaseItem? If the functionality for menu only?

Reply:
`This functionality we support only in menu widget. But it is not work correct if it moved to ui.menu.js file.`

Yes, it doesn't work because next menu levels is `subMenu` component that extends `contextMenu` component. So, this functionality should be on `menuBase` level for using both `menu` & `contextMenu`